### DISCUSSION TITLE
Feature request: add reverse proxy support

### DISCUSSION THREAD

**Discussion by MajliTech** [dani-garcia/vaultwarden#3523] [Ideas]
When I tried to access my instance with /vaultwarden, the resources are trying to get from /app, not /vaultwarden/app

**Comment by BlackDex** [comment]
Have you seen https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples ??

**Reply by BlackDex** [reply]
And https://github.com/dani-garcia/vaultwarden/wiki/Using-an-alternate-base-dir ??

**Comment by MajliTech** [comment]
Will check it, thanks
